### PR TITLE
New Alerts Guide

### PR DESCRIPTION
**Pull request by adnanrahic**
I've separated the Alerts and Events Guides. I'll work on the Events Guide on a separate branch.

### CODE REVIEW

File: docs/guide/alerts-guide.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+title: Sematext Alerts User Guide
+description: Sematext Cloud Alerts give you a flexible centralized notification system that lets you manage Alert Rules and Alert Conditions across your whole stack. Use Events to correlate them with Metrics and/or Logs, and see what is happening with your infrastructure.  
+
+Receiving Alerts when your software is misbehaving is crucial for every developer. Good alerting practices include:
```
Comment (otisg) [inline docs/guide/alerts-guide.md]: misbehaving => misbehaving or underperforming

Comment (adnanrahic) [reply docs/guide/alerts-guide.md]: Changed

**Comment by otisg** [comment]
@adnanrahic made more comments...

File: docs/guide/alerts-guide.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- **Thresholds** defined for **Alerts** when selected **metrics reach certain severity levels**
+- **Notification channels** through which the responsible team will be notified about the incident
+
+[Sematext Alerts](../alerts) help you address underlying issues before they affect user experience. There are three types of Alerts. 
+
+- **Threshold** - Alerts that are based on classic thresholds. They are **triggered** when something **crosses a pre-defined threshold**.
+- **Anomaly** - Alerts based on statistical anomaly detection. They are **triggered** when **values** suddenly **change and deviate from the baseline**. 
+- **Heartbeat** - Alerts **triggered** when **something you are monitoring**, like your servers, containers, or your applications, **stops sending data** to Sematext Cloud. 
+
+> **Threshold** and **Anomaly** Alerts can be triggered for both Monitoring and Logs Apps, while **Heartbeat** Alerts are only available for Monitoring Apps.
```
[the remark this answers is not in the document]
Comment (adnanrahic) [reply docs/guide/alerts-guide.md]: I spoke with Pawel. We will surely have threshold alerts for RUM, and heartbeat makes sense to have. But, nothing is surely defined and implemented in the UI yet. I'll add this info once we have a clearer picture about it.

**Pull request merged**